static/extensions/MubiLop/penguingpt.js

```diff
@@ -0,0 +1,752 @@
+                        }]
+                    },
+
+                    igModels: {
+                        acceptReporters: true,
+                        items: [
+                        {
+                            text: "DALL-E 3",
+                            value: "dalle-3"
+                        }, 
+                        {
+                            text: "Midjourney",
+                            value: "midjourney"
+                        },
+                        {
+                            text: "OpenJourney V4",
+                            value: "openjourney-v4"
+                        }, 
+                        {
+                            text: "Dreamshaper 8",
+                            value: "dreamshaper-8"
+                        }, 
+                        {
+                            text: "Anything V5",
+                            value: "anything-v5"
+                        }, 
+                        {
+                            text: "Realistic Vision V5",
+                            value: "realistic-vision-v5"
+                        }]
+                    },
+
+                    reqModels: {
+                        acceptReporters: true,
+                        items: [
+                        {
+                            text: "GPT 3.5 Turbo",
+                            value: "gpt-3.5-turbo"
+                        }, 
+                        {
+                            text: "GPT 4",
+                            value: "gpt-4"
+                        }, 
+                        {
+                            text: "GPT 4 1066 Preview (Recent)",
+                            value: "gpt-4-1106-preview"
+                        }, 
+                        {
+                            text: "GPT 4 0125 Preview (Turbo)",
+                            value: "gpt-4-0125-preview"
+                        }, 
+                        {
+                            text: "Llama 2",
+                            value: "llama-2-7b"
+                        }]
+                    }
+
+                }
+            };
+        }
+
+        getPrompt(args) {
+            if (args.TYPE !== '(select a prompt)') {
+                return args.TYPE;
+            } else {
+                return '';
+            }
+        }
+
+        setModel(args) {
+            this.model = args.MODEL
+        }
+
+        setApiUrl(args) {
+            const newApiUrl = args.URL;
+            // Update the api_url variable
+            api_url = newApiUrl;
+        }
+
+        moderate(args) {
+            const prompt = args.PROMPT;
+
+            return Scratch.fetch(`https://reverse.mubi.tech/v1/moderations`, {
```

Comment:
- Does not respect `api_url` variable, either disable the ability to change it or use the variable here
- Also doesnt even exist on the server? 404 error, remove this block or add it to the default api

Reply:
Moderate block has been deleted